packages/pigment-css-react/src/sx.js

```diff
@@ -1,7 +1,25 @@
 export default function sx(transformedSx, { className, style }) {
-  const sxClass = typeof transformedSx === 'string' ? transformedSx : transformedSx?.className;
-  const sxVars =
-    transformedSx && typeof transformedSx !== 'string' ? transformedSx.vars : undefined;
+  let sxClass = '';
+  let sxVars = {};
+
+  function iterateSx(element) {
+    if (element) {
+      sxClass += `${typeof element === 'string' ? element : element.className} `;
```

Comment:
```
sxClass = clsx(sxClass, typeof element === 'string' ? element : element.className)
```

This'll also take care of the extra space. Or you can `trim()` manually at the end.

Reply:
Added `.trim()`